src/test/java/com/google/api/codegen/grpcmetadatagen/PackageMetadataGeneratorTest.java

```diff
@@ -79,17 +79,26 @@ protected Object run() throws Exception {
         getTestDataLocator().findTestData("fakeprotodir").getPath());
     options.set(GrpcMetadataGenerator.METADATA_CONFIG_FILE, metadataConfigPath);
     options.set(GrpcMetadataGenerator.LANGUAGE, language);
-    Map<String, Doc> generatedDocs = new GrpcMetadataGenerator(options).generateDocs(model);
-    OutputCollector collector = new OutputCollector(Paths.get(outFile));
-    Files.walkFileTree(Paths.get(outFile), collector);
-    return new ImmutableMap.Builder<String, Doc>()
-        .putAll(generatedDocs)
-        .putAll(collector.getResults())
-        .build();
+    Map<String, Doc> generatedDocs = new GrpcMetadataGenerator(options).generate(model);
+
+    if (language == "java") {
+      return new ImmutableMap.Builder<String, Doc>().putAll(generatedDocs).build();
+    } else {
```

Comment:
Let's make Python the `if` case here, since it seems to be the only one that will use a Copier (which would necessitate the use of a Collector to collect the copied files). All other languages can just use the `<String, Doc>` map.